components/cards/SuggestionCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { MusicalNoteIcon } from "@heroicons/react/24/solid";
+import Image, { StaticImageData } from "next/image";
+import { IUser } from "@/interfaces/User";
+
+export interface Instrument {
+    instrument: string
+    icon: { readonly default: StaticImageData }
+}
+
+export interface Role {
+    instrument: Instrument
+    filledBy: IUser | undefined
+}
```

Comment:
Is it possible to use the IRole interface for this?

Reply:
The IRole interface contains a reference to the instrument objects, like "SINGER", whereas the Role interface contains an instruments obj like  { instrument: "Singer", icon: require("public/assets/instruments/icons8-micro-50.png") } because I don't want to save that data in the db and bc i need to render the images beforehand